index.js

```diff
@@ -0,0 +1,194 @@
+import {
+  startGame,
+  gameBord,
+  replayGame,
+  allcells,
+  nextPlayer,
+  playersForm,
+  playerOneName,
+  playerTwoName,
+  alertMsg,
+  gameOver,
+  // eslint-disable-next-line import/extensions
+} from './dom.js';
+
+// eslint-disable-next-line func-names
+const domElements = (function () {
+  const inAlertMsg = (msg) => {
+    alertMsg.innerHTML = msg;
+  };
+
+  const inNextPlayer = (msg) => {
+    nextPlayer.innerHTML = msg;
+  };
+
+  const inGameOver = (msg) => {
+    gameOver.innerHTML = msg;
+  };
+
+  const clickedBtn = (el, sym) => {
+    el.innerHTML = sym;
+  };
+
+  const resetBtns = (i) => {
+    allcells[i].innerHTML = '';
+  };
+
+  return {
+    inAlertMsg,
+    inNextPlayer,
+    inGameOver,
+    clickedBtn,
+    resetBtns,
+  };
+}());
+
+// eslint-disable-next-line func-names
+const gameBordModule = (function () {
+  const gameOn = false;
+  const playerOneName = '';
+  const playerTwoName = '';
+  const currentPlayerName = '';
+  const currentPlayerSym = '';
+  const gameBoard = ['', '', '', '', '', '', '', '', ''];
+  const count = 0;
+
+  return {
+    gameOn,
+    playerOneName,
+    playerTwoName,
+    currentPlayerName,
+    currentPlayerSym,
+    gameBoard,
+    count,
+  };
+}());
+
+// eslint-disable-next-line func-names
+const gameModule = (function () {
+  const gameInit = (playerOneName, playerTwoName) => {
+    gameBordModule.gameOn = true;
+    gameBordModule.playerOneName = playerOneName;
+    gameBordModule.playerTwoName = playerTwoName;
+    gameBordModule.currentPlayerName = gameBordModule.playerOneName;
+    gameBordModule.currentPlayerSym = 'X';
+    playersForm.style.display = 'none';
+    domElements.inNextPlayer(`${gameBordModule.currentPlayerName} Start!`);
+  };
+
+  const winChecker = () => {
+    let win = false;
+    const gb = gameBordModule.gameBoard;
+    const winCombinations = [
+      [gb[0], gb[1], gb[2]],
+      [gb[3], gb[4], gb[5]],
+      [gb[6], gb[7], gb[8]],
+      [gb[0], gb[3], gb[6]],
+      [gb[1], gb[4], gb[7]],
+      [gb[2], gb[5], gb[8]],
+      [gb[0], gb[4], gb[8]],
+      [gb[2], gb[4], gb[6]],
+    ];
+    const checkSym = (sym) => sym === gameBordModule.currentPlayerSym;
+    winCombinations.forEach((el) => {
+      if (el.every(checkSym)) {
+        win = true;
+      }
+    });
+    return win;
+  };
+
+  const drawChecker = () => {
+    if (gameBordModule.count > 8) {
+      return true;
+    }
+    return false;
+  };
+
+  const checkNextMove = () => {
+    if (winChecker()) {
+      domElements.inGameOver(`${gameBordModule.currentPlayerName} Wins!`);
+      gameOver.style.display = 'block';
```

Comment:
- [ ] All styles should be in the CSS file and then the dom should be manipulated by togling a classlist on dom.js.